src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -619,4 +619,10 @@
   <data name="PX1076Title" xml:space="preserve">
     <value>This code calls Acumatica internal API marked with PXInternalUseOnlyAttribute which is not intended for public use</value>
   </data>
+  <data name="PX1056TitleIsActive" xml:space="preserve">
```

Comment:
@EAndrosova please review diagnostic messages

Reply:
@EAndrosova the new diagnostic for review is actually PX1056:
`A PXGraph instance cannot be initialized inside the IsActive method`
and 
`A PXGraph instance cannot be initialized inside the IsActiveForGraph&lt;TGraph&gt; method`

The one you mentioned is for already released diagnostic so it was reviewed. The main purpose of that diagnostic is to prevent external developers from using our internal API, reserved by Acumatica for its own use. It targets external developers only. I think it's worth to clarify for them which company reserves the API since in general case there could be internal APIs from other companies like Microsoft. Plus it is already released. Still we can change it if needed.